index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>js quiz</title>
+</head>
+<body onkeyup="enter(event)">
+<div>
+<input type="text" id="char" name="char">Character
+<input type="text" id="height" name="height">Height
+<input type="button" onclick="grow()" value="grow">
```

Comment:
inline onClick functions are not ideal (until we get to Angular).  try using click events instead.  We will be talking about them in class tonight if you are unfamiliar with them.